app/src/components/Contexts/config.js

```diff
@@ -18,23 +22,35 @@ export const NETWORKS = {
         chainId : 42161
     }
 }
+
+// Contracts deployed by @ericyoondotcom with account 0xdCafA75DCAED81cba4155706E9d666112950E854 on 2023-01-26 via deploy script.
```

Comment:
Need a check to validate account that each of these contracts was created from.  Also you dont need a comment for the script as the contracts are visible on etherscan.